ui/public/javascript/editForm.js

```diff
@@ -0,0 +1,144 @@
+$( document ).ready(function() {
+    //guess ingredient quantities when field empty
+    $('.ingredient__detail__quantity', this).each(function() {
+        var quant = $(this).val()
+        if(quant === "") {
+            var re = /\d+/
+            var parsedIngredient = $(this).siblings(".ingredients__detail__parsed-ingredient").text()
+            var quantityGuess = parsedIngredient.match(re)
+            if(quantityGuess) {
+                $(this).val(quantityGuess[0])
+                $(this).attr("value", quantityGuess[0])
+            }
+        }
+    })
+
+    //guess ingredient unit when field empty
+    $(".ingredients__detail__unit", this).each(function(){
+        var unit = $(this).val()
+        if(unit === "") {
```

Comment:
Note, this wont be working at the moment as the form is setting everything to 'g' by default. TODO add empty string option (this field is an option).